repository/Seaside-Pharo-Tools-Web.package/WAScreenshot.class/instance/form.st

```diff
@@ -1,5 +1,5 @@
 accessing
 form
 	^ zoom
-		ifFalse: [ Display copy ]
+		ifFalse: [ UIManager default world imageForm ]
```

Comment:
This was more or less trial and error. I'm not sure this is actually the right way.

Reply:
If it works, it works